test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1636,6 +1636,44 @@ def forward(self, x):
         self.assertTrue(torch.ops.aten.clone.default in targets)
         self.assertTrue(torch.ops.aten.native_dropout.default not in targets)
 
+    def test_bn_move_exported_model_to_eval(self):
+        class M(torch.nn.Module):
+            def __init__(
+                self,
+            ):
+                super().__init__()
+                self.conv = torch.nn.Conv2d(3, 3, 3)
+                self.bn1 = torch.nn.BatchNorm2d(3)
+                self.bn2 = torch.nn.BatchNorm2d(3)
```

Comment:
Just wanted to understand this better. This test would still pass even if we only have 1 batch norm right? Since we're not doing `_fold_conv_bn_qat` here?

Reply:
Yes, this UT still pass if we only have 1 BN.

Reply:
Update the UT to only keep the first BN.